apps/server/src/modules/tool/context-external-tool/service/tool-version-service.ts

```diff
@@ -28,8 +28,13 @@ export class ToolVersionService {
 			const configurationStatus: ContextExternalToolConfigurationStatus = new ContextExternalToolConfigurationStatus({
 				isOutdatedOnScopeContext: false,
 				isOutdatedOnScopeSchool: false,
+				isDeactivated: false,
 			});
 
+			if (externalTool.isDeactivated || (schoolExternalTool.status && schoolExternalTool.status.isDeactivated)) {
```

Comment:
You can move this to a private function call it directly in the creation of the ContextExternalToolConfigurationStatus